src/main.rs

```diff
@@ -86,11 +90,26 @@ impl Drop for Deno {
 }
 
 fn main() {
-  log::set_max_level(log::LevelFilter::Debug);
+  let args = App::new("deno")
+    .version("0.0.0")
+    .about("A secure TypeScript runtime built on V8")
+    .author("Ryan D. <ry@tinyclouds.org>")
+    .arg(Arg::with_name("V8ARGS").required(true).multiple(true))
+    .arg(
+      Arg::with_name("debug")
+        .short("d")
+        .long("debug")
+        .help("Sets the log level to max"),
+    )
+    .get_matches();
+
+  if args.is_present("debug") {
+    log::set_max_level(log::LevelFilter::Debug);
```

Comment:
Does this work for you? You should get more output.. maybe this isn't the right call?

Reply:
It doesn't output more regardless of the branch. Not sure. Haven't used the log crate unfortunately. Can look into it.